wallet/connect/index.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Connect to MetaMask using various options.
+---
+
+# Connect to MetaMask
+
+You can connect your dapp to MetaMask using one of the following methods:
+
+- The [MetaMask SDK](metamask-sdk/index.md)
+- Third-party libraries with SDK support, e.g., [Wagmi](3rd-party-libraries/wagmi.md) or
```

Comment:
```suggestion
- Third-party libraries with SDK support, such as [Wagmi](3rd-party-libraries/wagmi.md) or
```

We follow the Microsoft Writing Style Guide and refrain from using "e.g.": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg